<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewImg</class>
 <widget class="QDialog" name="DialogNewImg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New image</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Heigth:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBoxNewImgWidth">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Width:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>px</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBoxNewImgHeight">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QPushButton" name="pushButtonNewImgOk">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>px</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit">
     <property name="text">
      <string>New image</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QPushButton" name="pushButtonNewImgCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonNewImgOk</sender>
   <signal>clicked()</signal>
   <receiver>DialogNewImg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNewImgCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogNewImg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
